torch/onnx/utils.py

```diff
@@ -1663,6 +1667,15 @@ def _export(
     return torch_out
 
 
+@_beartype.beartype
+def _add_onnxscript_fn(model_proto: bytes) -> bytes:
+    model = onnx.load_model_from_string(model_proto)
+    if _custom_onnxscript is not None:
+        model.functions.extend(_custom_onnxscript)
```

Comment:
Discussed offline, we should update the logic to include only the function proto used in the model.